docs/en/classic/compute-admin-guide/configure/permissions.adoc

```diff
@@ -1,18 +1,18 @@
 [#permissions]
 == Permissions by Feature
 
-When you set up Prisma Cloud Compute to secure your cloud workloads, you'll need to ensure you've granted Prisma Cloud the right permissions.
+When you set up Prisma Cloud Compute to secure your cloud workloads, ensure you've granted Prisma Cloud the right permissions.
 The following tables list the permissions required for each of Compute's protection capabilities.
 
 Review the following sets of permissions to ensure Prisma Cloud has the right permissions.
```

Comment:
Delete 
Review the following sets of permissions to ensure Prisma Cloud has the right permissions.